cargo-shuttle/src/main.rs

```diff
@@ -4,8 +4,6 @@ use clap::Parser;
 
 #[tokio::main]
 async fn main() -> Result<()> {
-    env_logger::init();
```

Comment:
Shouldn't the fmt subscriber be registered here?

Reply:
I am still not certain.  I have a //TODO marker under it to find out, because the docs are not overly helpful either way.  I'm still working through e2e test fixes and expect to see something exploding eventually